XbSymbolDatabase.c

```diff
@@ -106,55 +106,57 @@ typedef const struct _SymbolDatabaseList {
 
 SymbolDatabaseList SymbolDBList[] = {
     // Support inline functions in .text section
-    { XbSymbolLib_D3D8,{ Sec_text, Sec_D3D, Sec_FLASHROM }, D3D8_OOVPAV2, D3D8_OOVPA_COUNT },
+    { XbSymbolLib_D3D8 | XbSymbolLib_D3D8LTCG,{ Sec_text, Sec_D3D, Sec_FLASHROM }, D3D8_OOVPA, D3D8_OOVPA_COUNT },
 
-    // Cannot support LTCG in HLE
-    { XbSymbolLib_D3D8LTCG,{ Sec_D3D }, D3D8LTCG_OOVPAV2, D3D8LTCG_OOVPA_COUNT },
+    // NOTE: May will not have full support to scan every symbols.
```

Comment:
`May not have` is a bit vague an expression. Perhaps say:
`This libraries contains LCTG-specific OOVPA not present in the main D3D library, which this is an extension of.`
or something along those lines.

Reply:
```
    // NOTE: LTCG is a compiler optimization extension which may not be found in standard ___ library.
    //       Not every LTCG symbols can be reliable to create signatures depending on the situation.
```

After made a 3rd draft, I believe should be enough to clarify the situation may occur any time in the future. If not, I'm willing to hear what the note should say.

Reply:
I had drop the notes about the LTCG comment from this pull request. It should be documented in a Wiki than repeating same notes in the source code.